Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionAutomatedALMDefects.cs

```diff
@@ -39,9 +39,10 @@ public enum eDefectsOpeningMode
 
         public new static class Fields
         {
-            public static string SelectedDefectsProfileID = "SelectedDefectsProfileID";
-            public static string DefectsOpeningModeForAll = "DefectsOpeningModeForAll";
-            public static string DefectsOpeningModeForMarked = "DefectsOpeningModeForMarked";
+            public const string SelectedDefectsProfileID = "SelectedDefectsProfileID";
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)